### ISSUE TITLE
How do I set only square ratio image configuration

### ISSUE THREAD

**Issue by faustinogagneten** [guoyingtao/Mantis#2]
The library has the possibility to add custom ratios and thats perfect. However, I only need to use square ratio. How do I remove the other default configurations? Does the library provides any configuration?

Thanks.

**Comment by guoyingtao** [comment]
Thanks for your suggestion! This is a good idea. I will add this feature in the next version.

**Comment by guoyingtao** [comment]
Just released version 0.28. Now you can set ratioOptions of the Mantis config to only keep you custom ratios. 
```swift
var config = Mantis.Config()
config.ratioOptions = [.custom]
//If you only want to use square ratio
config.ratioOptions = [.square]
`

**Comment by guoyingtao** [comment]
@SwiftyFlow 
Mantis's code changed a lot since 0.28. Can you try the code below

```Swift
var config = Mantis.Config()
config.presetFixedRatioType = .alwaysUsingOnePresetFixedRatio(ratio: 1.0)
```
